packages/teuchos/comm/src/Teuchos_Comm.hpp

```diff
@@ -145,7 +145,7 @@ class CommRequest : public Teuchos::Describable {
   /// complete the canceled request (as required by the MPI standard).
   virtual ~CommRequest() {}
 
-  //! Wait on this request (a blocking operation).
+  /// Wait on this request (a blocking operation).
```

Comment:
@pyt-viper Does SWIG not like `//!` Doxygen markers?

Reply:
SWIG likes them fine.  I was just experimenting...  I still get a warning that looks like this:

    /Users/williamspotz/Development/Trilinos/packages/teuchos/comm/src/Teuchos_Comm.hpp:152: Warning 566: Unexpected iterator value in DoxygenParser::parse

I think `DoxygenParser` is an internal SWIG variable.  It's not a helpful message.

Reply:
Thanks for explaining! :-D